src/misc.c

```diff
@@ -282,6 +285,39 @@ static JSValue tjs_print(JSContext *ctx, JSValueConst this_val, int argc, JSValu
     return JS_UNDEFINED;
 }
 
+static JSValue tjs_prompt(JSContext *ctx, JSValueConst this_val, int argc, JSValueConst *argv) {
+    char *str;
+
+    const char *message = "";
+    const char *default_value = "";
+
+    if (argc > 0) {
```

Comment:
you can drop this check, QuickJS guarantees that it will fill the number of arguments you set when defining the function (see my last comment)

Reply:
Both arguments are optional.